src/indent-alignment.js

```diff
@@ -75,6 +75,27 @@ module.exports = {
          );
       });
 
+      // Ensure top-level lists adhere to the start_indent setting if it`s defined
+      iterate(params.parsers.micromark.tokens, [ 'listOrdered', 'listUnordered' ], (token) => {
+         if (params.config.start_indent !== undefined) {
```

Comment:
Minor perf note: Rather than looping through every top-level list item and then deciding if we should do anything, what about doing the inverse? i.e. only loop though the top-level list items when `start_indent` is defined?